server/presenters/collection.js

```diff
@@ -34,8 +24,7 @@ export default function present(collection: Collection) {
   };
 
   if (collection.type === 'atlas') {
-    // Force alphabetical sorting
-    data.documents = sortDocuments(collection.documentStructure);
+    data.documents = collection.documentStructure;
```

Comment:
Need to make L23 `documents: []`, and then this something like:

```
if (collection.type === 'atlas' && collection.documentStructure) {
  data.documents = collection.documentStructure;
}
```

The sort that was here previously was converting the null to an empty array.

Reply:
@tommoor thanks for pointing on this. I wasn't aware of that behaviour. It should be resolved now in 660e968